cmd/dockerd/options.go

```diff
@@ -104,6 +105,7 @@ func (o *daemonOptions) installFlags(flags *pflag.FlagSet) {
 	flags.BoolVarP(&o.Debug, "debug", "D", false, "Enable debug mode")
 	flags.BoolVar(&o.Validate, "validate", false, "Validate daemon configuration and exit")
 	flags.StringVarP(&o.LogLevel, "log-level", "l", "info", `Set the logging level ("debug"|"info"|"warn"|"error"|"fatal")`)
+	flags.StringVarP(&o.LogFormat, "log-format", "", "text", `Set the logging format ("text"|"json")`)
```

Comment:
You can use `StringVar` here (instead of `StringVarP`), as we don't define a single-letter shorthand.

```diff
-	flags.StringVarP(&o.LogFormat, "log-format", "", "text", `Set the logging format ("text"|"json")`)
+	flags.StringVar(&o.LogFormat, "log-format", "text", `Set the logging format ("text"|"json")`)
```

Reply:
Fixed.